### ISSUE TITLE
Handle EntropyRequest out of band in hw-js library

### ISSUE THREAD

**Issue by olemis** [skycoin/hardware-wallet-js#120]
**Describe the bug**
Handle out-of-band entropy requests.

**Environment information:**

- OS: all
- Platform: discovered on `Linux name 4.19.0-1-amd64 1 SMP Debian 4.19.12-1 (2018-12-22) x86_64 GNU/Linux`
- Firmware Version: skycoin/hardware-wallet@453e780 

**Steps to Reproduce**

TBD , but should happen if Skywallet blocks interaction for more than 2 min  e.g. waiting for button press event.

**Actual behavior**
This is a sample error message displayed while working on #110 

```
Skiping button confirmation! MessageType_EntropyRequest
(node:45924) UnhandledPromiseRejectionWarning: Error: decodeFailureAndPinCode failed
    at devButtonRequestCallback (/PATH/TO/skycoin/hardware-wallet-js/device-wallet.js:1139:22)
    at devButtonRequestCallback (/PATH/TO/skycoin/hardware-wallet-js/device-wallet.js:714:5)
    at wordAckLoop (/PATH/TO/skycoin/hardware-wallet-js/device-wallet.js:1134:13)
    at wordAckCallback (/PATH/TO/skycoin/hardware-wallet-js/device-wallet.js:1117:5)
    at deviceHandle.read (/PATH/TO/skycoin/hardware-wallet-js/device-wallet.js:1108:13)
    at BufferReceiver.receiveBuffer (/PATH/TO/skycoin/hardware-wallet-js/buffer-receiver.js:50:7)
    at devHandleCallback (/PATH/TO/skycoin/hardware-wallet-js/device-wallet.js:169:26)
(node:45924) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:45924) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Expected behavior**
`EntropyRequest` should be handled silently by sending `EntropyAck` back to the device.

**Additional context**
See skycoin/hardware-wallet-go#127 , a similar issue for hw-go library

**Possible implementation**
TBD

**Comment by stdevStark** [comment]
I have this same problem too with `devRecoveryDevice`

```
Device Open
Writing a buffer of length  64 to the device
Received data of length:  64
That message says its size is:  2
msgid slice: Uint8Array [ 0, 26 ]
Slice: Uint8Array [ 0, 0, 0, 2 ]
Received header  msg this.kind:  MessageType_ButtonRequest  size:  2 buffer lenght:  64
Remaining bytesToGet: -53
Device Close
Device Open
Writing a buffer of length  64 to the device
Received data of length:  64
That message says its size is:  2
msgid slice: Uint8Array [ 0, 46 ]
Slice: Uint8Array [ 0, 0, 0, 2 ]
Received header  msg this.kind:  MessageType_WordRequest  size:  2 buffer lenght:  64
Remaining bytesToGet: -53
Device Close
Button Loop operation completed
Device Open
Going into WordAck loop
Device Close
Device Open
Inside wordReader callback, please input word:
barrel
Writing a buffer of length  64 to the device
Received data of length:  64
That message says its size is:  2
msgid slice: Uint8Array [ 0, 46 ]
Slice: Uint8Array [ 0, 0, 0, 2 ]
Received header  msg this.kind:  MessageType_WordRequest  size:  2 buffer lenght:  64
Remaining bytesToGet: -53
Going into WordAck loop
Device Close
Device Open
Inside wordReader callback, please input word:
minor
Writing a buffer of length  64 to the device
Received data of length:  64
That message says its size is:  2
msgid slice: Uint8Array [ 0, 46 ]
Slice: Uint8Array [ 0, 0, 0, 2 ]
Received header  msg this.kind:  MessageType_WordRequest  size:  2 buffer lenght:  64
Remaining bytesToGet: -53
Going into WordAck loop
Device Close
Device Open
Inside wordReader callback, please input word:
normal
Writing a buffer of length  64 to the device
Received data of length:  64
That message says its size is:  2
msgid slice: Uint8Array [ 0, 46 ]
Slice: Uint8Array [ 0, 0, 0, 2 ]
Received header  msg this.kind:  MessageType_WordRequest  size:  2 buffer lenght:  64
Remaining bytesToGet: -53
Going into WordAck loop
Device Close
Device Open
Inside wordReader callback, please input word:
trash
Writing a buffer of length  64 to the device
Received data of length:  64
That message says its size is:  2
msgid slice: Uint8Array [ 0, 46 ]
Slice: Uint8Array [ 0, 0, 0, 2 ]
Received header  msg this.kind:  MessageType_WordRequest  size:  2 buffer lenght:  64
Remaining bytesToGet: -53
Going into WordAck loop
Device Close
Device Open
Inside wordReader callback, please input word:
assume
Writing a buffer of length  64 to the device
Received data of length:  64
That message says its size is:  2
msgid slice: Uint8Array [ 0, 46 ]
Slice: Uint8Array [ 0, 0, 0, 2 ]
Received header  msg this.kind:  MessageType_WordRequest  size:  2 buffer lenght:  64
Remaining bytesToGet: -53
Going into WordAck loop
Device Close
Device Open
Inside wordReader callback, please input word:
claw
Writing a buffer of length  64 to the device
Received data of length:  64
That message says its size is:  2
msgid slice: Uint8Array [ 0, 46 ]
Slice: Uint8Array [ 0, 0, 0, 2 ]
Received header  msg this.kind:  MessageType_WordRequest  size:  2 buffer lenght:  64
Remaining bytesToGet: -53
Going into WordAck loop
Device Close
Device Open
Inside wordReader callback, please input word:
moment
Writing a buffer of length  64 to the device
Received data of length:  64
That message says its size is:  2
msgid slice: Uint8Array [ 0, 46 ]
Slice: Uint8Array [ 0, 0, 0, 2 ]
Received header  msg this.kind:  MessageType_WordRequest  size:  2 buffer lenght:  64
Remaining bytesToGet: -53
Going into WordAck loop
Device Close
Device Open
Inside wordReader callback, please input word:
course
Writing a buffer of length  64 to the device
Received data of length:  64
That message says its size is:  0
msgid slice: Uint8Array [ 0, 35 ]
Slice: Uint8Array [ 0, 0, 0, 0 ]
Skiping message parsing, msgSize == 0
Device Close
Skiping button confirmation! MessageType_EntropyRequest
(node:11685) UnhandledPromiseRejectionWarning: Error: decodeFailureAndPinCode failed
    at devButtonRequestCallback (/go/src/github.com/skycoin/hardware-wallet-js/skywallet.js:1105:22)
    at devButtonRequestCallback (/go/src/github.com/skycoin/hardware-wallet-js/skywallet.js:693:5)
    at wordAckLoop (/go/src/github.com/skycoin/hardware-wallet-js/skywallet.js:1100:13)
    at wordAckCallback (/go/src/github.com/skycoin/hardware-wallet-js/skywallet.js:1083:5)
    at deviceHandle.read (/go/src/github.com/skycoin/hardware-wallet-js/skywallet.js:1076:11)
    at BufferReceiver.receiveBuffer (/go/src/github.com/skycoin/hardware-wallet-js/buffer-receiver.js:50:7)
    at devHandleCallback (/go/src/github.com/skycoin/hardware-wallet-js/skywallet.js:247:24)
(node:11685) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:11685) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

```

**Comment by stdevStark** [comment]
I was doing #112

**Comment by stdevStark** [comment]
@olemis after b8e1c5b this issue is seen no more

**Comment by olemis** [comment]
> @olemis after [b8e1c5b](https://github.com/skycoin/hardware-wallet-js/commit/b8e1c5b409058f78f90010cccb0a2662f7795fc6) this issue is seen no more

Yes , I could go beyond recovery in pre-release procedure this time . Will be closed by #117
